bank-accounts.rb

```diff
@@ -6,13 +6,15 @@
 module Bank
 
   class Account
+    @@min_balance = 0
```

Comment:
I like that you used a separate variable for this! It's great to keep it separate